src/main/webapp/view/lecture-view.css

```diff
@@ -128,6 +132,10 @@ discussion-comment[highlighted] > [slot='header'] {
   padding: 15px;
 }
 
+#transcript-container .spinner-border {
+  margin-left: 40%;
```

Comment:
nit: check if margin auto works to center horizontally here

Reply:
(resolved in chat) margin-auto unfortunately doesn't work :(